src/entry.cpp

```diff
@@ -43,15 +43,11 @@ POSSIBILITY OF SUCH DAMAGE.
 #include "libtorrent/entry.hpp"
 #include "libtorrent/hex.hpp"
 
-namespace
-{
-}
-
 namespace libtorrent
 {
 	namespace detail
 	{
-		TORRENT_EXPORT char const* integer_to_str(char* buf, int size
```

Comment:
true, about to remove it (btw, it bothers me that this function is only used in `bencode` but declared an defined in `entry`)